### PR TITLE
Add documentation for upgrading PostgreSQL database

### PR DESCRIPTION
**Pull request by Martchus**
See https://progress.opensuse.org/issues/76924#note-10

### CODE REVIEW

**Review by perlpunk — APPROVED** [review]

**Review by kraih — APPROVED** [review]

File: docs/Installing.asciidoc [hunk 1]
```diff
@@ -299,6 +299,144 @@ For older versions of openQA, you can migrate from SQLite to PostgreSQL
...
+openSUSE's PostgreSQL and openQA packaging but with a little adaption they can
+likely be used for other setups as well. These instructions can migrate big
+databases in seconds without requiring additional disk space. However, services
+need to be stopped during the (short) migration.
+
+1. Locate the `data`-directory. Its path is configured in
+`/etc/sysconfig/postgresql` and should be `/var/lib/pgsql/data` by default. The
+paths in the next steps assume the default.
+
+2. To ease migrations, I would recommend making the `data`-directory a symlink
```
Comment (okurz) [inline docs/Installing.asciidoc:314]: Who is the *I* here? I suggest to write instead s/I would recommend/it is recommended/

File: docs/Installing.asciidoc [hunk 2]
```diff
@@ -299,6 +299,144 @@ For older versions of openQA, you can migrate from SQLite to PostgreSQL
...
+to a versioned directory. So the file system layout would look like this:
++
+[source,sh]
+----
+martchus@ariel:~> sudo -u postgres ls -l /var/lib/pgsql | grep data
```
Comment (okurz) [inline docs/Installing.asciidoc:319]: This could be non-personalized

File: docs/Installing.asciidoc [hunk 3]
```diff
@@ -299,6 +299,144 @@ For older versions of openQA, you can migrate from SQLite to PostgreSQL
 according to
 <<Pitfalls.asciidoc#db-migration,DB migration from SQLite to PostgreSQL>>
 
+==== Migrate PostgreSQL database on openSUSE
```
Comment (okurz) [inline docs/Installing.asciidoc:302]: This is now a section in the chapter "Basic configuration", better we move it to "Advanced".

**Review by okurz — COMMENTED** [review]
Implementation of my suggested changes: https://github.com/os-autoinst/openQA/pull/4193

File: docs/Installing.asciidoc (hunk 3 above)
Comment (Martchus) [reply docs/Installing.asciidoc:302]: Ok, point taken. I'd only like to note that the steps are not optional anymore if we decide to use newer PostgreSQL features at some point (unless one doesn't care about the data).

**Pull request merged**